spec/node/asciidoctor.spec.js

```diff
@@ -141,6 +141,8 @@ describe('Node.js', function () {
       expect(blocks[1].getBlocks()[1].getId()).toBe('blockid');
       expect(blocks[1].getBlocks()[1].getStyle()).toBe('quote');
       expect(blocks[1].getBlocks()[1].getAttribute('attribution')).toBe('Abraham Lincoln');
+      expect(blocks[1].getBlocks()[1].getSourceLines()).toEqual(['This is a quote.', 'It has a title, id, and attribution.'])
```

Comment:
Missing semicolon 😉 
At some point we may switch to https://standardjs.com/ and remove semicolons but in the meantime...